package.json

```diff
@@ -56,7 +56,7 @@
     }
   },
   "dependencies": {
-    "co-busboy": "^1.4.0",
+    "co-busboy": "^1.4.1",
```

Comment:
`^1.4.0` means `>=1.4.0 && < 2.0.0`, so not need to change here.

Reply:
- https://docs.npmjs.com/cli/v6/configuring-npm/package-json#dependencies
- https://semver.org/lang/zh-CN/